packages/discord.js/src/structures/Guild.js

```diff
@@ -748,11 +748,11 @@ class Guild extends AnonymousGuild {
    * @property {DefaultMessageNotificationLevel|number} [defaultMessageNotifications] The default message notification
```

Comment:
This is also nullable. It resets the default message notification level back to all messages (0).